tests/baselines/reference/ClassAndModuleThatMergeWithModuleMemberThatUsesClassTypeParameter.types

```diff
@@ -15,7 +15,7 @@ module clodule1 {
 >clodule1 : typeof clodule1
 
     function f(x: T) { }
->f : (x: any) => void
+>f : (x: T) => void

```

Comment:
I get it, but this seems like a problem when the type isn't resolved.

Reply:
Hmmm, assuming this is resolving to the `errorType`, I can try seeing how it is if I remap the node to `any` if we find that type, but still preserve the whitespace/comments. (But it's worth mentioning that we don't do declaration emit when there's errors like this (unless they're suppressed) so this is mostly quickinfo behavior documented in this .types baseline)